src/JuMPContainer.jl

```diff
@@ -99,6 +99,7 @@ end
 
 pushmeta!(x::JuMPContainer, sym::Symbol, val) = (x.meta[sym] = val)
 getmeta(x::JuMPContainer, sym::Symbol) = x.meta[sym]
+hasmeta(x::JuMPContainer, sym::Symbol) = haskey(x.meta, sym)
```

Comment:
Great, thanks (I didn't mean to imply you should do that work, just that it shouldn't be forgotten)